### PR TITLE
Task-863330  UG documentation for stacked line and 100 stacked line series in .NET MAUI

### PR DESCRIPTION
**Pull request by Harsha-SF4223**
### Description ###

Added the UG document for .NET MAUI stacked line and stacked line 100 series.

### CODE REVIEW

File: MAUI/Cartesian-Charts/StackedLine100.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    chart.Series.Add(series2);     
+    chart.Series.Add(series3); 
+    chart.Series.Add(series4); 
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Stacking Line 100 Chart in .NET MAUI Cartesian Charts.](chart-types-images\net-maui-cartesian-charts-stacked-line-100-chart.png)
```
Comment (NanthiniMahalingam) [inline MAUI/Cartesian-Charts/StackedLine100.md:105]: @Harsha-SF4223 ,
change the image for stacked line and stacked 100 line series.

Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/StackedLine100.md:105]: I have changed the images for stacked line and stacked 100 line

File: MAUI/Cartesian-Charts/StackedLine.md [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Dashed Stacked Line chart in MAUI](Chart-types-images/maui_dashed_stacked_line_chart.png)
+
+## Enable Marker
+
+A marker, also known as a symbol, is used to determine or highlight the position of the data point. To enable markers in the series, set the [ShowMarkers](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.StackingLineSeries.html#Syncfusion_Maui_Charts_StackingLineSeries_ShowMarkers) property to `true`.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/StackedLine.md:221]: The StackingLineSeries has not been published live yet. Why is the API link included here? How does it work?

File: MAUI/Cartesian-Charts/StackedLine.md [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Stacked Line Marker support in MAUI Chart](Chart-types-images/stacked_line_marker_support.png)
+
+### Marker customization
+
+In order to change the series markers appearance, create an instance of the [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.StackingLineSeries.html#Syncfusion_Maui_Charts_StackingLineSeries_MarkerSettings) property. The following properties are used to customize marker appearance.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/StackedLine.md:310]: API link is wrongly updated here

File: MAUI/Cartesian-Charts/StackedLine.md [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+    chart.Series.Add(series3); 
+    chart.Series.Add(series4); 
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Stacked Line Custom Marker support in MAUI Chart](Chart-types-images/stacked_line_custom_marker_support.png)
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/StackedLine.md:445]: Remove the image for marker customization

File: MAUI/Cartesian-Charts/StackedLine.md [hunk 5]
```diff
@@ -0,0 +1,445 @@
...
+            <chart:StackingLineSeries.MarkerSettings>
+                <chart:ChartMarkerSettings Type="Diamond" 
+                                           StrokeWidth="2"
+                                           Fill="White"
+                                           Height="15"
+                                           Width="10"/>
+            </chart:StackingLineSeries.MarkerSettings>    
+        </chart:StackingLineSeries>   
+    
+        <chart:StackingLineSeries ItemsSource="{Binding Data2}"
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/StackedLine.md:342]: add single series like this
image

File: MAUI/Cartesian-Charts/StackedLine100.md [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+control: SfCartesianChart
+documentation: ug
+---
+
+# Stacked Line 100 Chart in .NET MAUI Chart
+
+
+The stacked line 100 chart is a type of chart that displays multiple line series stacked on top of each other, to indicate the cumulative value of the data points at 100%.
+
+To render a stacked line 100 chart, create an instance of the [StackingLine100Series]() and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of the [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/StackedLine100.md:15]: address the all stacked line md file comments for stacked 100 line md too

**Review by SaiyathAliFathima — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/StackedLine.md [hunk 7]
```diff
@@ -0,0 +1,445 @@
...
+title: Stacked Line Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about stacked line chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Stacked Line Chart in .NET MAUI Chart
+
+
```
[the remark this answers is not in the document]
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/StackedLine.md:12]: removed spaces

File: MAUI/Cartesian-Charts/StackedLine.md [hunk 8]
```diff
@@ -0,0 +1,445 @@
...
+{% endhighlight %}
+
+{% endtabs %}
+
+![Stacking Line Chart in .NET MAUI Cartesian Charts.](chart-types-images\net-maui-cartesian-charts-stacked-line-chart.png)
+
+### Dashed Stacked Line
+
+The [StrokeDashArray](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.StackingLineSeries.html#Syncfusion_Maui_Charts_StackingLine_StrokeDashArray) property of [StackingLineSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.StackingLineSeries.html?tabs=tabid-1) is used to render the stacked line series with dashes. Odd value is considered as rendering size and even value is considered as gap.
```
[the remark this answers is not in the document]
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/StackedLine.md:110]: removed link

File: MAUI/Cartesian-Charts/StackedLine.md (hunk 2 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/StackedLine.md:221]: removed link

File: MAUI/Cartesian-Charts/StackedLine.md (hunk 3 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/StackedLine.md:310]: Removed link

File: MAUI/Cartesian-Charts/StackedLine.md (hunk 5 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/StackedLine.md:342]: modified code

File: MAUI/Cartesian-Charts/StackedLine.md (hunk 4 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/StackedLine.md:445]: Removed image

File: MAUI/Cartesian-Charts/StackedLine100.md (hunk 6 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/StackedLine100.md:15]: modified stacked line 100 based on stacked line suggestions

File: maui-toc.html [hunk 9]
```diff
@@ -329,6 +329,8 @@
                     <li><a href="/maui/cartesian-charts/BarChart">Bar Chart</a></li>
                     <li><a href="/maui/cartesian-charts/line">Line Chart</a></li>
                     <li><a href="/maui/cartesian-charts/StepLine">StepLine Chart</a></li>
+                    <li><a href="/maui/cartesian-charts/StackedLine">Stacked Line Chart</a></li>
```
[the remark this answers is not in the document]
Comment (Harsha-SF4223) [reply maui-toc.html]: updated

**Review by SaiyathAliFathima — APPROVED** [review]
File changes are fine

**Pull request merged**